src/libs/API/parameters/UpdatePolicyRoomNameParams.ts

```diff
@@ -1,6 +1,7 @@
 type UpdatePolicyRoomNameParams = {
     reportID: string;
     policyRoomName: string;
+    renamedRoomReportActionID: string;
```

Comment:
The type is number

Reply:
Thanks

Reply:
This won't be compatible with the root type:

https://github.com/Expensify/App/blob/a3d4f7c5d348a861f1af44f0037c2c01f7a7912d/src/types/onyx/ReportAction.ts#L103

And other similar params:

https://github.com/Expensify/App/blob/ea5c6417cedb2638407151c2c710fb3d69c706f8/src/libs/API/parameters/CreateTaskParams.ts#L5

Actually, we use string for the `reportActionID` across the whole project:
<img width="930" alt="image" src="https://github.com/Expensify/App/assets/12595293/cbf3bf5b-a499-4c97-b10a-1df690615a9e">

Reply:
Okay fine! let's keep it as string